### DISCUSSION TITLE
Should dynamic triple/double buffering be removed?

### DISCUSSION THREAD

**Discussion by puxplaying** [puxplaying/mutter-x11-scaling#36] [Polls]
GNOME 44 will not include MR [1441](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1441) as it isn't considered ready yet upstream.
On the other hand [Ubuntu](https://salsa.debian.org/gnome-team/mutter/-/blob/ubuntu/master/debian/patches/debian/Support-Dynamic-triple-double-buffering.patch) and [Debian](https://salsa.debian.org/gnome-team/mutter/-/merge_requests/88) ship it by default.

**Comment by virgilnr** [comment]
Better to keep it shipped. I love your patch because it also make the system animation felt snappier. I don't see the reason to remove it. It makes gnome perform better with no significant drawback.
